src/lambdaisland/uri.cljc

```diff
@@ -3,33 +3,41 @@
   (:require [clojure.string :as str]
             [lambdaisland.uri.normalize :as normalize])
   #?(:clj (:import clojure.lang.IFn)))
-
+str
```

Comment:
What does this here do?

Reply:
It's sacrificial, for religious purposes only.

Reply:
amen to that!